Makefile

```diff
@@ -84,10 +78,10 @@ setup-git-config:
 setup-git: ensure-venv setup-git-config
 	@echo "--> Installing git hooks"
 	mkdir -p .git/hooks && cd .git/hooks && ln -sf ../../config/hooks/* ./
-	@PYENV_VERSION=$(REQUIRED_PY3_VERSION) python3 -c '' || (echo 'Please run `make setup-pyenv` to install the required Python 3 version.'; exit 1)
+	@python3 -c '' || (echo 'Please run `make setup-pyenv` to install the required Python 3 version.'; exit 1)
 	@# pre-commit loosely pins virtualenv, which has caused problems in the past.
 	$(PIP) install "pre-commit==1.18.2" "virtualenv==20.0.32"
```

Comment:
Note, I'll be removing the `"virtualenv==20.0.32"` pin in a future PR which will upgrade pre-commit along with black.

Reply:
Looking forward to it! It feels nice we're getting to clean things up.